platform/Span.h

```diff
@@ -49,29 +49,29 @@ class is_convertible
 
 /**
  * Special value for the Extent parameter of Span.
- * If the type use this value then the size of the array is stored in the object
+ * If the type uses this value, then the size of the array is stored in the object
  * at runtime.
  */
 #define SPAN_DYNAMIC_EXTENT -1
 
 /**
- * Non owning view to a sequence of contiguous elements.
+ * Nonowning view to a sequence of contiguous elements.
```

Comment:
Shouldn't it be Non-owning ?

Reply:
Use hyphens when the last letter of the suffix and the first letter of the prefix are the same [proto-oncology, non-negotiable, post-traumatic and so on]. There are some exceptions, such as when the suffix is a proper noun or an acronym. Then, use a hyphen regardless of the first letter of the suffix.

Reply:
That makes sense. Thanks Amanda. 
Looks like a lot of the CS literature spell it wrong.